Source/SIMPLib/DataArrays/DataArray.cpp

```diff
@@ -208,9 +208,6 @@ DataArray<T>::DataArray(size_t numTuples, const QString& name, const comp_dims_t
     m_Size = m_NumTuples * m_NumComponents;
     m_MaxId = (m_Size > 0) ? m_Size - 1 : m_Size;
   }
-#if 0
-      MUD_FLAP_0 = MUD_FLAP_1 = MUD_FLAP_2 = MUD_FLAP_3 = MUD_FLAP_4 = MUD_FLAP_5 = 0xABABABABABABABABul;
```

Comment:
This is handy to have in the code in case we are trying to figure out if somebody walked off the end of the array